From b6462a1268b7f8eb1c0b52d35c79555adbe78e93 Mon Sep 17 00:00:00 2001
From: Marko Saukko <marko.saukko@jolla.com>
Date: Thu, 18 Jan 2018 11:42:47 +0000
Subject: [PATCH] Add text/x-vnote, *.vnt.

The original patch is by Hannu Mallat, created to fix MER#1096 and JB#10854

Signed-off-by: Marko Saukko <marko.saukko@jolla.com>
---
 freedesktop.org.xml.in | 10 ++++++++++
 tests/list             |  3 +++
 tests/test.vnt         |  4 ++++
 3 files changed, 17 insertions(+)
 create mode 100644 tests/test.vnt

diff --git a/freedesktop.org.xml.in b/freedesktop.org.xml.in
index caa4202..0e873d3 100644
--- a/freedesktop.org.xml.in
+++ b/freedesktop.org.xml.in
@@ -7147,4 +7147,14 @@ command to generate the output files.
     </magic>
   </mime-type>
 
+  <mime-type type="text/x-vnote">
+    <_comment>Calendar/journal textual note</_comment>
+    <sub-class-of type="text/plain"/>
+    <magic priority="50">
+      <match type="string" value="BEGIN:VNOTE" offset="0"/>
+      <match type="string" value="begin:vnote" offset="0"/>
+    </magic>
+    <glob pattern="*.vnt"/>
+  </mime-type>
+
 </mime-info>
diff --git a/tests/list b/tests/list
index d527325..a488065 100644
--- a/tests/list
+++ b/tests/list
@@ -668,3 +668,6 @@ ascii.stl model/x.stl-ascii x
 binary.stl model/x.stl-ascii xxx
 ascii.stl model/x.stl-binary oxx
 test.gcode text/x.gcode ox
+
+# vNote (IrMC 1.1 specification)
+test.vnt text/x-vnote
diff --git a/tests/test.vnt b/tests/test.vnt
new file mode 100644
index 0000000..0978129
--- /dev/null
+++ b/tests/test.vnt
@@ -0,0 +1,4 @@
+BEGIN:VNOTE
+VERSION:1.1
+BODY: Test note
+END:VNOTE
-- 
1.8.3-rc3

